prow/plugins/label/label.go

```diff
@@ -183,6 +199,11 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 		}
 	}
 
+	if len(sigMatches) == 0 {
```

Comment:
Maybe guard the call below directly?

    toRepeat := []string{}
    if len(sigMatches) > 0 {
        isMember, _ := gc.IsMember(ae.org, ae.login)
        if !isMember {
            toRepeat = ae.getRepeats(sigMatches, existingLabels)
        }
    }